### PR TITLE
Replace native inputs with NcTextfield

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Part of #7708 
* Minor visual changes (screenshots attached)

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
image | image
image | image
image | image

### 🚧 Tasks

- [ ] Code review
- [ ] Visual check

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not required
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: src/components/SettingsDialog/SettingsDialog.vue [hunk 1]
```diff
@@ -37,11 +37,16 @@
...
-				:disabled="attachmentFolderLoading"
-				@click="selectAttachmentFolder">
+			<div class="app-settings-section__wrapper">
+				<NcTextField class="app-settings-section__input"
+					:value="attachmentFolder"
+					:disabled="true"
+					@click="selectAttachmentFolder" />
+				<NcButton type="primary"
+					@click="selectAttachmentFolder">
+					{{ t('spreed', 'Browse...') }}
```
Comment (nickvergessen) [inline src/components/SettingsDialog/SettingsDialog.vue:47]:
```suggestion
					{{ t('spreed', 'Browse…') }}
```

File: src/components/SetGuestUsername.vue [hunk 2]
```diff
@@ -79,12 +75,12 @@ export default {
...
-			return this.$store.getters.getDisplayName()
+			return this.$store.getters.getDisplayName() || t('spreed', 'Guest')
 		},
 		displayNameLabel() {
-			return t('spreed', 'Display name: <strong>{name}</strong>', {
-				name: this.actorDisplayName ? this.actorDisplayName : t('spreed', 'Guest'),
-			})
+			return t('spreed', 'Display name: {name}', {
+				name: `<strong>${escapeHtml(this.actorDisplayName)}</strong>`,
+			}, undefined, { escape: false })
```
Comment (ShGKme) [inline src/components/SetGuestUsername.vue:83]: Why use `escapeHtml` here?

File: src/components/SetGuestUsername.vue [hunk 3]
```diff
@@ -152,7 +148,7 @@ export default {
 			this.isEditingUsername = !this.isEditingUsername
 			if (this.isEditingUsername) {
 				this.$nextTick(() => {
-					this.$refs.usernameInput.focus()
+					document.getElementById('guest_username_input').focus()
```
[the remark this answers is not in the document]
Comment (ShGKme) [reply src/components/SetGuestUsername.vue:151]: Yes, it should be with `$refs`. It doesn't looks good, but we should add a method to the `Nextcloud/vue`.

File: src/components/SetGuestUsername.vue (hunk 2 above)
Comment (Antreesy) [reply src/components/SetGuestUsername.vue:83]: See reference:
https://github.com/nextcloud/spreed/pull/9344#discussion_r1173639124

File: src/components/BreakoutRoomsEditor/BreakoutRoomsEditor.vue [hunk 4]
```diff
@@ -137,6 +140,10 @@ export default {
 				console.debug(error)
 			}
 		},
+
+		setAmount(event) {
+			this.amount = Number(event.target.value)
+		},
```
[the remark this answers is not in the document]
Comment (Antreesy) [reply src/components/BreakoutRoomsEditor/BreakoutRoomsEditor.vue:146]: value type should be a String for this component, and @input returns an event. So I'm not sure we could use a v-model here until it fixed upstream

[the remark this answers is not in the document]
Comment (ShGKme) [reply src/components/BreakoutRoomsEditor/BreakoutRoomsEditor.vue:148]: I mean, previously it was `parseFloat` inside `.number`, not `Number`. It has a bit different behavior
image

[the remark this answers is not in the document]
Comment (Antreesy) [reply src/components/BreakoutRoomsEditor/BreakoutRoomsEditor.vue]: Fixed

**Comment by Antreesy** [comment]
Rebased, amend all fixups into one commit

**Review by ShGKme — APPROVED** [review]

**Pull request merged**